runtime/tests/tests.rs

```diff
@@ -416,7 +416,7 @@ fn staking_validators_should_receive_equal_transaction_fee_reward() {
 				// Check tx fee reward went to the stash account of validator
 				assert_eq!(
 					<GenericAsset as MultiCurrency>::free_balance(&stash, Some(CENTRAPAY_ASSET_ID)),
-					balance_amount + per_fee_reward + per_staking_reward
+					balance_amount + per_fee_reward + per_staking_reward - 1600
```

Comment:
@holygits do we know why this value has changed?

Reply:
not 💯 sure on this. It will need to be resolved in the follow up PR